book/src/prefabs/prefabs_technical_explanation.md

```diff
@@ -0,0 +1,323 @@
+# Prefabs Technical Explanation
+
+A `Prefab` in Amethyst is at the core a simple list of future entities, where each entry in
+the list consists of two pieces of optional data:
+
+* a parent index that refers to a different entry in the list
+* a data collection implementing the trait `PrefabData`
+
+To instantiate a `Prefab`, we put a `Handle<Prefab<T>>` on an `Entity`. The `Entity` we put
+the `Handle` on is referred to as the main `Entity`, and the first entry in the list inside a
+`Prefab` refers to this `Entity`. All other entries in the list will spawn a new `Entity` on
```

Comment:
```suggestion
`Prefab` will be applied to this `Entity`. All other entries in the list will spawn a new `Entity` on
```
I didn't understand on first read and had to check the documentation. What about this?

Reply:
Oh, this was a pre existing file that I renamed (no content was changed), I'm planning to break it up into separate "how to" guides in the coming weeks